### ISSUE TITLE
Litecoin & Ethereum support?

### ISSUE THREAD

**Issue by briannyeko** [bitpay/copay#4234]
When will you enable Copay support with Litecoin & Ethereum?

**Comment by dabura667** [comment]
There's already a fork for that, I think.

**Comment by martindale** [comment]
Having support for [the Asset Issuance Element](https://www.elementsproject.org/elements/asset-issuance/) would be nice, actually.

**Comment by beatdigger** [comment]
Can someone explain this in layman terms?
What is a "fork" and how do I implement it?
How can a simpleton like myslef add my Litecoin to my CoPay?

**Comment by dabura667** [comment]
> How can a simpleton like myslef add my Litecoin to my CoPay?

You can't.

You have to go find an app that copy + pasted Copay and changed it to support Litecoin etc.

**Comment by kaptkrunch** [comment]
Wait, bitpay can deal with LTC?

What the hell!

**Comment by eabz** [comment]
Take a look at PolisPay it suppers Litecoin. Ethereum is on the way

**Comment by kaptkrunch** [comment]
will do thanks!
   On Saturday, September 1, 2018, 12:28:49 PM CDT, Cronos <notifications@github.com> wrote:  
 
 
Take a look at PolisPay it suppers Litecoin. Ethereum is on the way

**Comment by martindale** [comment]
Ethereum is larger than Bcash, so why the preferential treatment?  🤔
